### DISCUSSION TITLE
Why the AcpuCache.php has been deleted?

### DISCUSSION THREAD

**Discussion by benjaminhenriest** [doctrine/doctrine2#10926] [Support Questions] [answered]
I was updating a project symfony version while I noticed that ApcuCache has been deleted from the project, I tried to find the reason but without success. 

Does someone could enlighten me on this please?

**Comment by derrabus** [comment, marked as answer]
https://www.doctrine-project.org/projects/doctrine-cache/en/2.2/index.html#deprecation-notice

**Comment by benjaminhenriest** [comment]
Thank you for the fast answer @derrabus !

Yet I still see some references to these cache drivers in the code, is this deprecated code or maybe some people find a way to install it?

https://github.com/doctrine/orm/blob/17500f56eaa930f5cd14d765bc2cd851c7d37cc0/lib/Doctrine/ORM/Tools/Setup.php#L240C11-L242C10

**Reply by derrabus** [reply]
Yes, this is deprecated code.

https://github.com/doctrine/orm/blob/17500f56eaa930f5cd14d765bc2cd851c7d37cc0/lib/Doctrine/ORM/Tools/Setup.php#L41

The ORM does not attempt use these classes if they don't exist anymore.

**Reply by benjaminhenriest** [reply]
Nice thank you for your time!
